frontend/src/app/utils/FabricUtils.ts

```diff
@@ -69,6 +69,7 @@ export class FabricUtils {
     return DEFAULT_TAGS.map((tag) => {
       const fullTag: Tag = {
         boardID,
+        tagID: tag?.tagID,
```

Comment:
Once you make it required, you can also remove the ? from here.

Reply:
Ye will do that, it was giving and error before but now i realize its because i need to updat ethe DEFAULT_TAGS so that they include the tagID as well.